HTTPServer.driver/Driver.cs

```diff
@@ -32,6 +32,7 @@ public static void Main(string[] args)
             commandDetails.Add(Tuple.Create((ICriteria)new VersionNotSupportedCriteria(), (IHttpHandler)new VersionNotSupported()));
             commandDetails.Add(Tuple.Create((ICriteria)new ContentsCriteria(), (IHttpHandler)new GetContents(pathContents)));
             commandDetails.Add(Tuple.Create((ICriteria)new PostCriteria(), (IHttpHandler)new PostContents(pathContents)));
+            commandDetails.Add(Tuple.Create((ICriteria)new PutCriteria(), (IHttpHandler)new PutContents(pathContents)));
```

Comment:
i'd much rather see this adding code as part of `RequestRouter`.  iow, i would want to be able to call something like `RequestRouter.AddHandler(criteria, handler)`.  This keeps the code outside of `RequestRouter` ignorant of details that really only matter to the router class.